app/src/main/kotlin/com/wire/android/util/FileManager.kt

```diff
@@ -109,12 +109,17 @@ class FileManager @Inject constructor(@ApplicationContext private val context: C
     suspend fun getAssetBundleFromUri(
         attachmentUri: Uri,
         tempCachePath: Path,
+        audioPath: Path? = null,
         dispatcher: DispatcherProvider = DefaultDispatcherProvider(),
     ): AssetBundle? = withContext(dispatcher.io()) {
         try {
-            val assetFileName = context.getFileName(attachmentUri) ?: throw IOException("The selected asset has an invalid name")
+            val assetFileName = context.getFileName(attachmentUri)
+                ?: throw IOException("The selected asset has an invalid name")
             val fullTempAssetPath = "$tempCachePath/${UUID.randomUUID()}".toPath()
-            val mimeType = attachmentUri.getMimeType(context).orDefault(DEFAULT_FILE_MIME_TYPE)
+            val assetPath = audioPath ?: fullTempAssetPath
+            val mimeType = if (audioPath != null) AUDIO_MIME_TYPE else attachmentUri
+                .getMimeType(context)
+                .orDefault(DEFAULT_FILE_MIME_TYPE)
```

Comment:
This part specifically, if we go by `getMimeType(..)..` it would render as a `VIDEO` file due to its current `mimeType` being `video/mp4` despite having only audio and no video data due to the way `MediaRecorder` creates the file.